CHANGELOG.md

```diff
@@ -1,6 +1,56 @@
 # Changelog
 
-## v0.10.x - TBD
+## v1.1.0 - TBD
```

Comment:
Would the next version be 1.0.1?

Reply:
Nevermind, my mistake.

Reply:
No.  Next version is a minor release. We do patch releases only for well, patches.